xbmc/TextureCacheJob.cpp

```diff
@@ -68,6 +68,36 @@ bool CTextureCacheJob::DoWork()
   return false;
 }
 
+namespace
+{
+// Most PVR images use "additional_info" to signify 'ownership' of basic images for easy
+// cache cleaning, rather than special generated images
+bool IsPVROwnedImage(std::string additional_info)
```

Comment:
please pass by const ref also for the other helper functions
```suggestion
bool IsPVROwnedImage(const std::string& additional_info)
```

Reply:
Thanks, all updated.